client/my-sites/plan-features-2023-grid/index.jsx

```diff
@@ -258,6 +263,52 @@ export class PlanFeatures2023Grid extends Component {
 			<PlanFeatures2023GridFeatures planProperties={ planProperties } domainName={ domainName } />
 		);
 	}
+
+	getFeatureToStorageMap( storageFeature ) {
```

Comment:
Minor nit: This method doesn't return a map but returns a translated string. Can we maybe rename this to `getStorageStringFromFeature` or similar?

Reply:
Done, pushed a change for this.